sql-plugin/src/main/scala/com/nvidia/spark/rapids/cudf_utils/HostConcatResultUtil.scala

```diff
@@ -28,8 +29,7 @@ object HostConcatResultUtil extends Arm {
    */
   def rowsOnlyHostConcatResult(numRows: Int): HostConcatResult = {
     new HostConcatResult(
-      new JCudfSerialization.SerializedTableHeader(
-        Array.empty, numRows, 0L),
+      new JCudfSerialization.SerializedTableHeader(numRows),
```

Comment:
This constructor is not accessible now that this file is in a different package

Reply:
Did you look at the dependency PR https://github.com/rapidsai/cudf/pull/10949